test/e2e/push_test.go

```diff
@@ -116,6 +116,15 @@ var _ = Describe("Podman push", func() {
 		push := podmanTest.Podman([]string{"push", "-q", "--tls-verify=false", "--remove-signatures", ALPINE, "localhost:5000/my-alpine"})
 		push.WaitWithDefaultTimeout()
 		Expect(push).Should(Exit(0))
+		Expect(len(push.ErrorToString())).To(Equal(0))
+
+		push = podmanTest.Podman([]string{"push", "--tls-verify=false", "--remove-signatures", ALPINE, "localhost:5000/my-alpine"})
+		push.WaitWithDefaultTimeout()
+		Expect(push).Should(Exit(0))
+		Expect(push.ErrorToString()).To(ContainSubstring("Copying blob "))
```

Comment:
just a small nit here, please copy ErrorToString result to a variable.
ErrorToString() or also OutputToString() copy the full buffer and use it with strings.Fields and strings.Join every time. I know that this is a bigger issue since basically all tests use it like this and I have no idea why this is implemented in this way

Reply:
I found it ugly as well and shrugged given the amount of tests that do that but it's a good idea to start :_)